.travis.yml

```diff
@@ -32,7 +32,9 @@ install:
   - npm install && npm run lint
 
 script:
-  - npm run build_spec && npm run test_mocha && node ./node_modules/markdown-doctest/bin/cmd.js
+  - npm install typescript@2.0.7 && npm run build_spec && npm run test_mocha
+  - npm install typescript@latest && npm run build_spec && npm run test_mocha
```

Comment:
as #2066 does not have near-term possibility to check in (and even for this), I am not worried too much since per each case this can be easily taken out as necessary.